### PR TITLE
[navigation/navigation_fixedwing.c] Fix Coding Style

### PR DESCRIPTION
**Pull request by JulioCesarMatias**

### CODE REVIEW

**Comment by Jetrell** [comment]
@JulioCesarMatias
I found `fw_loiter_direction = YAW` presently hit and miss in the way it operates. Some times it changes the loiter direction via Yaw stick input. And other times it doesn't.
Maybe this fix will make it more reliable.

File: src/main/navigation/navigation_fixedwing.c [hunk 1]
```diff
@@ -238,8 +238,14 @@ static int8_t loiterDirection(void) {
     int8_t dir = 1; //NAV_LOITER_RIGHT
     if (pidProfile()->loiter_direction == NAV_LOITER_LEFT) dir = -1;
     if (pidProfile()->loiter_direction == NAV_LOITER_YAW) {
-        if (rcCommand[YAW] < -250) loiterDirYaw = 1; //RIGHT //yaw is contrariwise
-        if (rcCommand[YAW] > 250) loiterDirYaw = -1; //LEFT  //see annexCode in fc_core.c
```
Comment (avsaase) [inline src/main/navigation/navigation_fixedwing.c:242]: I agree with everything else but I kinda like these one line if statements for very simple thing. I think they improve the readability of the code.

Comment (JulioCesarMatias) [reply src/main/navigation/navigation_fixedwing.c:242]: putting everything on one line won't make the algorithm any faster, I'm just following what's written here >> https://github.com/iNavFlight/inav/blob/master/docs/development/Development.md

Comment (JulioCesarMatias) [reply src/main/navigation/navigation_fixedwing.c:242]: I disagree to leave it like this

**Review by DzikuVx — APPROVED** [review]

**Pull request merged**